core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiServiceSpec.scala

```diff
@@ -114,9 +114,9 @@ class MethodConfigApiServiceSpec extends ApiServiceSpec with TestDriverComponent
       AgoraMethod("dsde", "good_and_bad", 1))
 
     val expectedSuccessInputs = Seq("goodAndBad.goodAndBadTask.good_in")
-    val expectedFailureInputs = Map("goodAndBad.goodAndBadTask.bad_in" -> "Failed at line 1, column 1: 'workspace.' expected but 'd' found")
+    val expectedFailureInputs = Map("goodAndBad.goodAndBadTask.bad_in" -> "Error while parsing the expression. Offending symbol is on line 1 at position 0. Error: mismatched input 'does' expecting {'{', '[', 'workspace.', 'this', 'true', 'false', 'null', STRING, NUMBER}")
```

Comment:
Note to all of us - depending on the decision about whether this change is acceptable, we will need to provide release notes describing the change.